### PR TITLE
ENG-2412 make link text require if page selected

### PR DESCRIPTION
**Pull request by ichalagashvili**
- if page link is selected, then default language link text should become mandatory

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ichalagashvili**

If they are not already assigned, you can assign the PR to them by writing `/assign @ichalagashvili` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-cms).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-cms/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/ui/widget-forms/ContentConfigFormBody.js
Comment (fedelocci) [inline src/ui/widget-forms/ContentConfigFormBody.js:343]: is not better to set the default to en?

File: src/ui/widget-forms/ContentsQueryConfig.js
Comment (fedelocci) [inline src/ui/widget-forms/ContentsQueryConfig.js:534]: like previous?

**Review by fedelocci — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-cms/pull/401#pullrequestreview-665137592" title="Approved">fedelocci</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-cms).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/entando-cms/blob/master/OWNERS)~~ [fedelocci]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
